app-modules/service-management/tests/Filament/ServiceMonitoring/Pages/CreateServiceMonitoringTest.php

```diff
@@ -139,3 +135,45 @@
     expect($serviceMonitoringTarget->teams())->exists()->toBeTrue();
     expect($serviceMonitoringTarget->users())->exists()->toBeTrue();
 });
+
+test('it will validate multiple valid forms of URL and IP Address', function (array $validUrls, array $invalidUrls) {
+    asSuperAdmin();
+
+    foreach ($validUrls as $url) {
+        $request = ServiceMonitoringTarget::factory()->make(['domain' => $url])->toArray();
+
+        livewire(CreateServiceMonitoring::class)
+            ->fillForm($request)
+            ->call('create')
+            ->assertHasNoFormErrors();
+    }
+
+    foreach ($invalidUrls as $url) {
+        $request = ServiceMonitoringTarget::factory()->make(['domain' => $url])->toArray();
+
+        livewire(CreateServiceMonitoring::class)
+            ->fillForm($request)
+            ->call('create')
+            ->assertHasFormErrors(['domain']);
+    }
+})
+    ->with([
+        [
+            'validUrls' => [
+                'http://example.com',
```

Comment:
I also mentioend that `example.com` should be valid as well in business expectation and also your regex would consider that valid as well. So please have that as a valid option.